recipes/moc/all/conanfile.py

```diff
@@ -0,0 +1,121 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from os import rename
+from os.path import isdir, join
+from shutil import move
+from glob import glob
+from distutils.version import LooseVersion
+
+class MocConan(ConanFile):
+    _noCmakeConfigFiles = True
+    _supported_compilers = [
+        ("Linux", "gcc", "6"),
+        ("Linux", "clang", "6"),
+        ("Macos", "gcc", "6"),
+        ("Macos", "clang", "6")
```

Comment:
and apple-clang?

Reply:
The MacOs build is failing on the build server but I'm unable to understand why given what minimal logging info I can see hence I'm disabling that configuration.